app/assets/stylesheets/variables/fonts.scss

```diff
@@ -1,4 +1,5 @@
 // .scss for multiline attributes
+@import url(http://fonts.googleapis.com/css?family=Open+Sans);
```

Comment:
If this is necessary, we should self host it and use HTTPS.

Reply:
I can test again, but without it I was getting the browser falling back to the base sans-serif font.